src/libraries/System.Drawing.Common/src/System/Drawing/Printing/InvalidPrinterException.Serializable.cs

```diff
@@ -2,7 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-// This file isn't built into the .csproj in corefx but is consumed by Mono.
+// This file isn't built into the .csproj at
+// https://github.com/dotnet/runtime/blob/master/src/libraries/System.Drawing.Common/src/System.Drawing.Common.csproj
```

Comment:
I don't think full urls make sense in any of these other files either.

Reply:
This is consistent with other files as my understanding is we want a functional link to the file,  not just a reference of it's location.  Keeping the full HTTP URL allows Visual Studio/Code to parse it as well.  This is the same reasoning for the links in the above .MD file above, it works in markdown, but Visual Studio as well.

Reply:
I think it is unlikely someone wants to click on this URL

Reply:
@danmosemsft that did cross my mind too. I'll change them all to a simpler rename. That should e;iminate @Tratcher concerns as well!